DDDEastAnglia.Tests/SponsorModelQueryBuilder.cs

```diff
@@ -0,0 +1,15 @@
+using DDDEastAnglia.Controllers;
+using DDDEastAnglia.DataAccess;
+
+namespace DDDEastAnglia.Tests
+{
+    public class SponsorModelQueryBuilder
```

Comment:
In fact, is it used? The `sponsor_list.context` appears to set this up manually.